app/models/infra_conversion_job.rb

```diff
@@ -442,13 +440,26 @@ def poll_transform_vm_complete
       raise migration_task.options[:virtv2v_message]
     when 'succeeded'
       update_migration_task_progress(:on_exit)
-      queue_signal(:poll_inventory_refresh_complete)
+      queue_signal(:inventory_refresh)
     end
   rescue StandardError => error
     update_migration_task_progress(:on_error)
     abort_conversion(error.message, 'error')
   end
 
+  def inventory_refresh
+    update_migration_task_progress(:on_entry)
+    if migration_task.options[:destination_vm_uuid].present?
+      target_collection = InventoryRefresh::TargetCollection.new(:manager => migration_task.destination_ems)
+      target_collection.add_target(:association => :vms, :manager_ref => {:ems_ref => migration_task.options[:destination_vm_uuid]})
+    end
```

Comment:
Once you have the `target` you still need to `task = EmsRefresh.queue_refresh_task(target)` (I assume you'll want to track the task for refresh completion) otherwise nothing is actually happening

Reply:
:+1: looks good